src/bindings/python/src/pyopenvino/graph/passes/pattern_ops.cpp

```diff
@@ -493,37 +493,36 @@ static void reg_pattern_optional(py::module m) {
                       }),
                       py::arg("type_name"),
                       R"(
-                      Create Optional with the given node type.
-
-                      :param type_names: node type. For example: ["opset8.Abs", "opset8.Relu"]
-                      :type type_names: List[str]
+        Create Optional with the given node type.
+        :param type_names: node type. For example: ["opset8.Abs", "opset8.Relu"]
+        :type type_names: List[str]
     )");
 
-    optional_type.def(py::init([](const std::vector<std::string>& type_names, const Predicate& predicate) {
-                          return std::make_shared<ov::pass::pattern::op::Optional>(get_types(type_names), predicate);
+    optional_type.def(py::init([](const std::vector<std::string>& type_names, const ov::Output<ov::Node>& input) {
+                          return std::make_shared<ov::pass::pattern::op::Optional>(get_types(type_names),
+                                                                                   ov::OutputVector{input},
+                                                                                   nullptr);
                       }),
                       py::arg("type_names"),
-                      py::arg("predicate"),
+                      py::arg("input"),
                       R"(
-                      Create Optional with the given node type and predicate.
+        Create Optional with the given node type and input node.
 
-                      :param type_names: node type. For example: ["opset8.Abs", "opset8.Relu"]
-                      :type type_names: List[str]
+        :param type_names: node type. For example: ["opset8.Abs", "opset8.Relu"]
+        :type type_names: List[str]
 
-                      :param predicate: Function that performs additional checks for matching.
-                      :type predicate: function
+        :param input: input node's output.
+        :type input: openvino.runtime.Output
     )");
 
-    optional_type.def(
-        py::init([](const std::vector<std::string>& type_names,
-                    const ov::Output<ov::Node>& input,
-                    const Predicate& predicate) {
-            return std::make_shared<ov::pass::pattern::op::Optional>(get_types(type_names), input, predicate);
-        }),
-        py::arg("type_names"),
-        py::arg("input"),
-        py::arg("predicate"),
-        R"(
+    optional_type.def(py::init([](const std::vector<std::string>& type_names, const std::shared_ptr<ov::Node>& input) {
+                          return std::make_shared<ov::pass::pattern::op::Optional>(get_types(type_names),
+                                                                                   ov::OutputVector{input},
+                                                                                   nullptr);
+                      }),
+                      py::arg("type_names"),
+                      py::arg("input"),
+                      R"(
```

Comment:
Here in the docstring still argument `predicate` is present.

Reply:
Why? Could you please explain? There is no predicate in `optional_type.def(py::init([](const std::vector<std::string>& type_names, const std::shared_ptr<ov::Node>& input) {`

Reply:
discussed online